tests/unit/amo/components/TestAddonMeta.js

```diff
@@ -114,6 +118,42 @@ describe(__filename, () => {
       });
 
       expect(getReviewCount(root).title).toEqual('No Ratings');
+
+      const averageMeta = root.find(MetadataCard).prop('metadata')[2];
+      expect(averageMeta.title).toEqual('Not rated yet');
+    });
+
+    it('renders the average rating', () => {
+      const average = 2.34;
+      const root = renderRatings({ average });
+
+      const averageMeta = root.find(MetadataCard).prop('metadata')[2];
```

Comment:
Getting the data with an index looks a bit fragile, maybe we could create a helper function to `getAverageMeta({ root })`?